tests/plugins/test_ddp_spawn_plugin.py

```diff
@@ -77,3 +78,22 @@ def test_ddp_spawn_extra_parameters(tmpdir):
     trainer.fit(model, datamodule=dm)
     assert trainer.callback_metrics[val_name] == torch.tensor(val)
     assert model.test_val == "test_val"
+
+
+class BoringModelDDP(BoringModel):
+    def on_train_start(self) -> None:
+        """Check model wrapped by DistributedDataParallel at configurate_ddp."""
+        assert isinstance(self.trainer.model, DistributedDataParallel)
+
+    def on_fit_end(self) -> None:
+        """Check teardown function at the end of training."""
+        assert self.trainer.model.device == torch.device("cpu")
+        assert isinstance(self.trainer.model, LightningModule)
+
+
+@RunIf(min_gpus=1)
```

Comment:
was the intention to test on gpu or cpu?
if no, remove this RunIf decorator, if yes, adjust the trainer arguments below.

Reply:
Good catch! Yeah, it intended to test GPU teardown, will add the param in trainer, thank!